plugins/base/src/main/kotlin/renderers/gfm/GfmRenderer.kt

```diff
@@ -0,0 +1,185 @@
+package org.jetbrains.dokka.base.renderers.gfm
+
+import org.jetbrains.dokka.base.renderers.DefaultRenderer
+import org.jetbrains.dokka.base.resolvers.DefaultLocationProvider
+import org.jetbrains.dokka.base.resolvers.LocationProviderFactory
+import org.jetbrains.dokka.pages.*
+import org.jetbrains.dokka.plugability.DokkaContext
+import java.lang.StringBuilder
+
+
+open class GfmRenderer(context: DokkaContext) : DefaultRenderer<StringBuilder>(context) {
+
+    override fun StringBuilder.buildHeader(level: Int, content: StringBuilder.() -> Unit) {
+        buildParagraph()
+        append("${"#".repeat(level)} ")
```

Comment:
```suggestion
        append("#".repeat(level) + " ")
```
better expresses the intention IMHO

Reply:
Ok